webdriver/tests/perform_actions/key_events.py

```diff
@@ -31,6 +31,9 @@ def test_modifier_key_sends_correct_events(session, key_reporter, key_chain, key
     code = ALL_EVENTS[event]["code"]
     value = ALL_EVENTS[event]["key"]
 
+    if (session.capabilities["browserName"] == 'internet explorer'):
```

Comment:
Style nits (sorry, but it will break the linter):

- Drop `(` and `)` for grouping.
- Use double quotes for the second string.

Reply:
Fixed in next commit. Also fixed it in the other place where the same construct was used.